ui_testing/pages/order_page.py

```diff
@@ -122,13 +127,33 @@ def create_new_order(self, material_type='', article='', contact='', test_plans=
         self.base_selenium.LOGGER.info(' Order created with no : {} '.format(order_no))
         return self.get_suborder_data()
 
+    def create_order_with_test_unit(self, material_type='', article='', contact='', test_units=[''],
```

Comment:
why we need to define this funiction in this PR?

Reply:
Done